samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java

```diff
@@ -87,9 +87,11 @@ public Boolean apply(TaskInstance t) {
     } else {
       Integer taskMaxConcurrency = config.getMaxConcurrency().getOrElse(defaultValue(1));
 
+      log.info("Got max messages in flight: " + taskMaxConcurrency);
```

Comment:
Not a big deal, but I'de add the word taskMaxConcurrency={}, to tie it better to the code. By just reading log 'messages in flight' might have different connotations.